src/cloud/benchflow/data-analyses-scheduler/scripts/sparkArgsConstructors.go

```diff
@@ -8,20 +8,22 @@ import (
 )
 
 // Function to resolve the correct configuration file path based on the SUT Name and Version
-func getConfigFilePath(SUTVersion string, SUTName string, fileName string) string {
+func getConfigFilePath(SUTVersion string, SUTName string, SUTType string, fileName string) string {
```

Comment:
@VincenzoFerme Sorry a commit didn't go through, now you should be able to see it